src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarArtists.tsx

```diff
@@ -24,11 +27,31 @@ export class ArtworkSidebarArtists extends React.Component<ArtistsProps> {
     )
   }
 
-  private renderSingleArtist(artist: Artist) {
+  private renderSingleArtist(artist: Artist, user, mediator) {
     return (
       <React.Fragment>
         {this.renderArtistName(artist)}
-        <FollowIcon is_followed={artist.is_followed} />
+        <FollowArtistButton
+          artist={artist}
+          user={user}
+          onOpenAuthModal={() => {
+            mediator.trigger("open:auth", {
+              mode: "signup",
+              copy: `Sign up to follow ${artist.name}`,
+              signupIntent: "follow artist",
+              afterSignUpAction: {
+                kind: "artist",
+                action: "follow",
+                objectId: artist.id,
+              },
+            })
+          }}
+          render={({ is_followed }) => {
+            return <FollowIcon is_followed={is_followed} />
```

Comment:
👍

Reply:
Can this be updated to follow normal js camelCase in the prop (`isFollowed`)? Or perhaps something a bit simpler like `<FollowIcon following>`.